src/components/myAccount/MyAccount.tsx

```diff
@@ -353,4 +393,20 @@ const useStyles = makeStyles()({
     lineHeight: '22px',
     color: theme.customPalette.colorActiveGrey,
   },
+  interests: {
+    display: 'flex',
+    justifyContent: 'center',
+    width: '100%',
```

Comment:
Do we really need this rule? By default, block elements have a width of 100% unless specified otherwise

Reply:
We’ll keep the styles as agreed earlier